examples/motoko_examples/minimal-counter-dapp/webpack.config.js

```diff
@@ -58,6 +62,9 @@ module.exports = {
         minimizer: [new TerserPlugin()]
     },
     resolve: {
+        alias: {
+            '∞': path.resolve('.dfx', network, 'canisters')
```

Comment:
I'm scared

Reply:
This was creating an alias to this character instead of copying all the files over. See 97f41cea671e981a85e215216995576af53955f4. After talking we decided not to do this so I've removed this in 9fe1d0e5cd6b4723120c4c31197d8b282b81f9b6